### ISSUE TITLE
wp_register_style and wp_register_scripts

### ISSUE THREAD

**Issue by emiluzelac** [CherryFramework/cherryframework4#49]
[../styles.php](https://github.com/CherryFramework/cherryframework4/blob/master/lib/functions/styles.php#L35)
[../scripts.php](https://github.com/CherryFramework/cherryframework4/blob/master/lib/functions/scripts.php#L81)

---

We will leave this one as question to @justintadlock since this part of codes are his ;)

Why do we need `wp_register_style` and `wp_register_scripts`? I understand that they don't load.

**Comment by justintadlock** [comment]
No,they don't load. They register a script for use. Looks good to me.

**Comment by emiluzelac** [comment]
Yep, got that part, the question was to what purpose this is, don't load them if we don't need to?
